test/setup/deployers/operator.ts

```diff
@@ -0,0 +1,33 @@
+import { IDeployer } from './types';
+import * as kubectl from '../../helpers/kubectl';
+
+export const operatorDeployer: IDeployer = {
+  deploy: deployKubernetesMonitor,
+};
+
+async function deployKubernetesMonitor(
+  integrationId: string,
+  _imageOpts: {
+    imageNameAndTag: string;
+    imagePullPolicy: string;
+  },
+): Promise<void> {
+  const namespace = 'snyk-monitor';
+  await kubectl.createNamespace(namespace);
+
+  const secretName = 'snyk-monitor';
+  const gcrDockercfg = process.env['GCR_IO_DOCKERCFG'] || '{}';
```

Comment:
this line seems familiar, maybe there's room for this to be consolidated somewhere?

Reply:
I think we can move it one level up, in setup.ts, because it's a prerequisite to deploying the monitor itself! It will be identical for every platform, regardless of the deployment type. Would you be OK if I did it in a separate PR? It's not a small change, because if I move this code then a lot of other code needs to be trimmed down (unused vars, the interface for the deploy() function gets affected...)?